theurgy.lic

```diff
@@ -167,7 +211,9 @@ until ['flawless performance to those on high', 'In your condition'].include? DR
   DRC.fix_standing
 end
 
-fput "get #{$settings.flint_lighter}", 'You get a', 'You are already'
+while checkright != "#{$settings.flint_lighter}"
+  fput "get #{$settings.flint_lighter}", 'You get a', 'You are already'
+end
 get_from_container('incense')
```

Comment:
The safer way to do this is to change `fput` to `bput`, but that requires you to capture some messaging.